### PR TITLE
feature/switch-template-cli-access

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Add support for cli Access for Switch::Template

NEWS file entries
=======================
Enhancements
-------------
* Add support for CLI Access for Switch::Template

Delete branch after merge
=========================
NO

Checklist
=========
- [ ] Document the feature
- [x] Add unit tests
- [ ] Add acceptance tests (TestLink)

### CODE REVIEW

File: html/pfappserver/root/static.alt/src/views/Configuration/_config/switchTemplate.js [hunk 1]
```diff
@@ -306,6 +306,48 @@ export const view = (form, meta = {}) => {
               }
             }
           ]
+        },
+        {
+          label: i18n.t('Cli Authorize Read Scope'),
```
Comment (extrafu) [inline html/pfappserver/root/static.alt/src/views/Configuration/_config/switchTemplate.js]: Cli -> CLI

File: html/pfappserver/root/static.alt/src/views/Configuration/_config/switchTemplate.js [hunk 2]
```diff
@@ -306,6 +306,48 @@ export const view = (form, meta = {}) => {
...
+                    valueLabel: i18n.t('Select value'),
+                    fields: radiusFields
+                  }
+                }
+              }
+            }
+          ]
+        },
+        {
+          label: i18n.t('Cli Authorize Write Scope'),
```
Comment (extrafu) [inline html/pfappserver/root/static.alt/src/views/Configuration/_config/switchTemplate.js]: Cli -> CLI

**Comment by jrouzierinverse** [comment]
bump

**Review by nqb — CHANGES_REQUESTED** [review]
It failed on my side with following error:

```
Sep 14 07:42:49 pfcen7dev packetfence_httpd.aaa: httpd.aaa(8717) ERROR: [mac:[undef]] Can't use string ("") as an ARRAY ref while "strict refs" in use at /usr/local/pf/lib/pf/radius/rest.pm line 33.
 (pf::WebAPI::REST::handler)
```
I got same error if I tested without a switch template (using Generic module)

**Comment by lzammit** [comment]
Works!

**Comment by nqb** [comment]
Need a rebase.

EDIT: I did it.

**Comment by extrafu** [comment]
Ready for merging? If so, let's do it.

**Comment by nqb** [comment]
Doing a latest test and I will merge.

**Pull request merged**